framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/Args/CommandLineArgumentParser.cs

```diff
@@ -40,6 +41,7 @@ public CommandLineArgs Parse(string[] args)
 
             if (!argumentList.Any())
             {
+                target = NamespaceHelper.NormalizeNamespace(target);
```

Comment:
This normalizing should be done only when 'target' must fit the project name regex. If you make this normalization here, any command which is not related to project name regex will be effected. 

So this normalization has to be moved to commands which takes a project name as 'target'.